components/authz-service/server/v2/policy_test.go

```diff
@@ -69,7 +69,7 @@ func TestCreatePolicy(t *testing.T) {
 			assert.Equal(t, []string{"cfgmgmt:nodes:*"}, resp.Statements[0].Actions)
 			assert.Equal(t, []string{"cfgmgmt:delete", "cfgmgmt:list"}, resp.Statements[0].Resources)
 			assert.Empty(t, resp.Statements[0].Role)
-			assert.Empty(t, resp.Statements[0].Projects)
+			assert.Equal(t, []string{constants_v2.AllProjectsExternalID}, resp.Statements[0].Projects)
```

Comment:
This affirms the change that no projects provided in 2.0 now is assigned a default "all projects".